scripts/authors.ts

```diff
@@ -162,6 +165,16 @@ namespace Commands {
                 console.log("=====================");
                 deduplicate(unknownAuthors).sort(sortAuthors).forEach(log);
             }
+
+
+            const allAuthors = deduplicate([...knownAuthors, ...unknownAuthors].map(a => a.split("<")[0])).sort(sortAuthors);

```

Comment:
If you split by `<` and pipe to a file, does that mean you'll end up with trailing whitespace on certain lines?

Reply:
I'll trim to make sure anyway, though this isn't pipeable because it's part of a longer list command